hvplot/plotting/core.py

```diff
@@ -632,17 +668,30 @@ def image(self, x=None, y=None, z=None, colorbar=True, **kwds):
         """
         Image plot
 
+        You can very often use `image` to display an x, y grid, like for example geographic
+        data with `latitude` and `longitude` coordinates.
+
+        Reference: https://hvplot.holoviz.org/reference/xarray/image.html
+
+        Example:
+
+        >>> import hvplot.xarray
+        >>> import xarray as xr
+        >>> ds = xr.tutorial.open_dataset('air_temperature')
+        >>> ds.hvplot.image(x='lon', y='lat', z='air', groupby='time', cmap='kbc_r')
+
         Parameters
         ----------
-        x, y : string, optional
```

Comment:
x,y does not display nicely in VS Code. It expects one argument. Not a comma separated list.